### PR TITLE
quickstart-validium Blockscout new UI screenshot

### PR DESCRIPTION
**Pull request by vbaranov**
Update quickstart-validium guide:
- change port for explorer new UI
- change screenshot of the explorer

### CODE REVIEW

File: docs/cdk/get-started/quickstart-validium.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ sudo make restart
 
 ## 3. Test validium
 
-3.1 Verify the block explorer is running by navigating to [localhost:4001](http://localhost:4001/).
+3.1 Verify the block explorer is running by navigating to [localhost](http://localhost/).
```
Comment (kmurphypolygon) [inline docs/cdk/get-started/quickstart-validium.md:143]: Hi @vbaranov I notice you have removed the port number. This number is mentioned a lot throughout the docs, particularly on setting up MetaMask. Can you confirm this should be removed here please? I will check on my side too.

Comment (vbaranov) [reply docs/cdk/get-started/quickstart-validium.md:143]: Yes, the new UI is running at 80 port when launching from quickstart-validium.

Comment (kmurphypolygon) [reply docs/cdk/get-started/quickstart-validium.md:143]: Thanks, does that mean you have to update the port number elsewhere in the doc?

Comment (vbaranov) [reply docs/cdk/get-started/quickstart-validium.md:143]: sure, let me check, where it should be updated and will make an additional commit.

Comment (kmurphypolygon) [reply docs/cdk/get-started/quickstart-validium.md:143]: Thanks, much appreciated :)

Comment (vbaranov) [reply docs/cdk/get-started/quickstart-validium.md:143]: @kmurphypolygon update in https://github.com/0xPolygon/polygon-docs/pull/235/commits/1797e30d3ab9af7f57b18d3b2402b96df2798022

**Comment by kmurphypolygon** [comment]
Thanks so much @vbaranov :)
Approving and merging now.
Will test in staging later on.

**Review by kmurphypolygon — APPROVED** [review]
LGTM

**Pull request merged**